docs/examples/spatial_transformation/histogram_matching.jl

```diff
@@ -0,0 +1,35 @@
+# ---
+# cover: assets/woman_darkhair.tif
+# title: Histogram Matching
+# author: Ashwani Rathee
+# date: 2020-11-21
+# ---
+
+# This demo demonstrates the feature of histogram matching.
+
+# In histogram matching,the pixels of source image is manipulated so that the histogram 
+# of reference image matches with source image's histogram.
+
+# Here,histogram matching is shown for gray scale image.But,it can also be applied on 
+# RGB images too.Histogram matching can be used to normalize two images, when the images 
+# were acquired at the same local illumination (such as shadows) over the same location,
+# but by different sensors,atmospheric conditions or global illumination. 
+
+using Images,ImageCore,TestImages,ImageContrastAdjustment,ImageView
+img_source=testimage("woman_darkhair")
+img_reference=testimage("lena_gray_512")
```

Comment:
Following the discussion in https://github.com/JuliaImages/TestImages.jl/issues/72, we don't use lena image in examples.

Reply:
Yeah,I already saw that one, but unfortunately, I couldn't decide on which one to use(there is a need for more test images in my opinion)...Which ones should I use?? ..I did try for outside RGB images which looked pretty good as examples(cat-source,cafe-reference,transformed one)